crates/meilisearch/src/main.rs

```diff
@@ -139,6 +146,27 @@ async fn try_main() -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Prompt the user about the contact email for support and news.
+/// It only displays the prompt if the input is an interactive terminal.
+async fn prompt_for_contact_email() -> anyhow::Result<Option<String>> {
+    let stdin = tokio::io::stdin();
+
+    if !stdin.is_terminal() {
+        return Ok(None);
+    }
+
+    println!("Would you mind providing your contact email for support and news?");
+    println!("We will use it to contact you with news only.");
+    print!("contact email> ");
```

Comment:
Hey @guimachiavelli, @dureuill, and @tpayet, I need a better text to prompt users for their email. Something legally better and friendlier than that. Can you help me with this part, please?

Reply:
Thank you for your help, I improved the prompting to use the one on the mini-dashboard.